code/game/atoms_movable.dm

```diff
@@ -58,6 +58,10 @@
 		pulledby = null
 	return ..()
 
+//Returns an atom's power cell, if it has one. Overload for individual items.
+/atom/movable/proc/get_cell()

```

Comment:
If you're going to do this, anything that has a cell needs to use this proc, as on TG.

Then we can port the power inducer, eventually.

Reply:
Understood. I have added get_cell() proc to the remaining items that are using power cells, hopefully I did not miss any.

Reply:
> Then we can port the power inducer, eventually.

For this also all the "bcell" must be replaced by "cell". 
I'm currently working on that for Hispania.